sequence_processing_pipeline/templates/compute_sequence_counts_for_normalization.sbatch

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -l
+#SBATCH -J {{job_name}}             # norm
+#SBATCH --time {{wall_time_limit}}  # 24:00:00
+#SBATCH --mem {{mem_in_gb}}G        # 8G
+#SBATCH -N {{node_count}}           # 1
+#SBATCH -c {{cores_per_task}}       # 1
+#SBATCH -p {{queue_name}}           # qiita
+
+#SBATCH --output {{output}}/logs/compute_sequence_counts_%x-%A_%a.out
+#SBATCH --error {{output}}/logs/compute_sequence_counts_%x-%A_%a.err
+
+# NB: output appears normal w/out.
+# source activate qiime2-2023.5
+
+set -x
+set -e
+set -o pipefail
+
+echo $TMPDIR
+
+mkdir -p {{output_path}}
+wc -l {{tellread_output}}/Full/*_I1_C5[0-9][0-9].fastq.gz.corrected.err_barcode_removed.fastq > {{output_path}}/record_counts.txt
```

Comment:
In the past I have seen that, in some occassions, `wc -l` is not the best to count fastq files; mainly when they are extra blank new lines or missing an ending line. I prefer to use `fqtools count`; however, I not sure if the change will be needed here. Thoughts?

Reply:
I agree. This is just what they were using in the original sbatch. I will be tweaking all of them during testing to use seqtk to count the sequences (in parallel where possible) instead. The reason being that we'll already have it installed to support metapool.